src/vivarium_gates_bep/globals.py

```diff
@@ -136,19 +138,14 @@ def confidence_interval_variance(upper, lower):
     }
 }
 
-CRUDE_BW_SHIFT = 147.71
-CRUDE_BW_SHIFT_UPPER = 210.90
-CRUDE_BW_SHIFT_LOWER = 84.52
```

Comment:
@alibow this is the Maternal BMI effect on birthweight, I think?

Reply:
Yeah, sorry I missed that!